launchers/basic/build.gradle.kts

```diff
@@ -1,45 +0,0 @@
-/*
```

Comment:
`launchers/basic/build.gradle.kts` still seems to be there, but empty. Just delete the entire directory :)

Reply:
No idea how this file got back in there again oO

Reply:
Should now be deleted